package.json

```diff
@@ -55,7 +55,8 @@
   },
   "resolutions": {
     "encoding": "0.1.13",
-    "expect-playwright": "0.8.0"
+    "expect-playwright": "0.8.0",
+    "vfile": "4.2.1"
```

Comment:
This is a peer dependency of Unified and minor version later as it includes the relevant polyfills for Webpack 5 support. Unified 10 is ESM only which we currently cannot support.